tools/flecstan/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+
+# CMake
+project(flecstan LANGUAGES CXX)
+cmake_minimum_required(VERSION 3.8) # probably not necessary
+
+# C++
+set(CMAKE_CXX_COMPILER g++)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread -fno-rtti")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pedantic -Wall -s")
+
+# Header file location
+include_directories(src)
+
+# Source files
+file(GLOB SOURCES "src/*.cc")
```

Comment:
Globing is bad, as new file won't get picked up with re-running CMake, please use an explicit list.

Reply:
Try it out, add a new `cc`, it won't get picked up without rerunning CMake, this tends to confuse people more than being helpful. Ask  @ddaniel so a background.